app/javascript/packs/application.js

```diff
@@ -2,4 +2,5 @@ require("@rails/ujs").start()
 require("turbolinks").start()
 require("@rails/activestorage").start()
 require("channels")
-require("bootstrap")
\ No newline at end of file
+require("bootstrap")
+require("bootstrap/dist/js/bootstrap")
```

Comment:
同じですので，こちらだけ残しておきましょうか。

```suggestion
require("bootstrap/dist/js/bootstrap")
```